xbmc/events/BaseEvent.cpp

```diff
@@ -20,6 +20,9 @@
 
 #include "BaseEvent.h"
 #include "guilib/LocalizeStrings.h"
+#include "utils/StringUtils.h"
+
+unsigned int CBaseEvent::s_nextIndex = 0;
```

Comment:
Cool, not loving my own solution very much, I will give std::chrono::steady_clock a try. Hopefully it is available and working on all Kodi platforms. Thanks for the input, @Paxxi